app/utils/prepareAnswer.js

```diff
@@ -131,17 +131,14 @@ function prepareTestingResourceAnswer(stateResources) {
   }
 
   const data = [`\n\n\u{1F9EA} COVID-19 Testing Labs in ${stateResources[0].state}, found ${stateResources.length} results\n`];
+  let tempData;
```

Comment:
Please move this inside the loop and then we are good to go. Just directly use it like `let tempData = bla bla`.

Reply:
okay doing it 👍